cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,26 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('should have a visible play button and duration', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media) {
+              const {
+                durationISO8601,
+              } = media.model.blocks[1].model.blocks[0].model.versions[0];
+
+              cy.get('div[class^="StyledVideoContainer"]').within(() => {
+                cy.get('button').should('be.visible');
+                cy.get('svg').should('be.visible');
```

Comment:
If we combine these by selecting the svg inside the button, we can assert both exist and are visible without making two selections. E.G.:
```
cy.get('button svg').should('be.visible');
```

Reply:
I've now amended this.